python/hopsworks/core/environment_api.py

```diff
@@ -57,21 +63,26 @@ def create_environment(self, await_creation=True):
             self._project_id,
             "python",
             "environments",
-            client.get_python_version(),
+            name,
         ]
         headers = {"content-type": "application/json"}
+        data = {"name": name,
+                "baseImage": {
+                    "name": base_environment_name,
```

Comment:
If you add an Enum for the available base environment names, we can validate the name before sending the request to the backend.

Reply:
The backend validates the name and sends a list of valid environment names back. I think adding the enum will come at a later stage maybe when I'm back.